fsd_test_utils/fixtures/db_fixtures.py

```diff
@@ -12,7 +12,7 @@ def prep_db(reuse_db=False):
     upgrade() is always run to make sure the schema is up to date.
     """
 
-    from config import Config
+    from pre_award.config import Config
```

Comment:
Do we need to check both paths? Is this used by any other repos, eg post-award or fab?

Reply:
Good spot, yes it seems FAB uses it too. Have allowed for both paths